assets/sass/06-components/footer-navigation.scss

```diff
@@ -0,0 +1,43 @@
+
+.footer-navigation {
+	font-size: var(--global--font-size-xs);
+	margin-top: calc(2 * var(--global--spacing-vertical));
+	text-align: center;
```

Comment:
With text-align: center, here, is 	`justify-content: center;` on the UL .footer-navigation-wrapper still needed?
I am not sure this is correct.

Reply:
When I removed the `justify-content: center;` it didn't align in the center anymore. But when I removed the `text-align: center;` and kept the other one it still aligned centered. So the question is if the `text-align: center` is needed.

Reply:
I made a small commit to remove it.